assets/queries/terraform/azure/aks_private_clusters_disabled/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "599318f2-6653-4569-9e21-041d06c63a89",
+  "queryName": "AKS Private Clusters Disabled",
```

change the directory name to `aks_private_cluster_disabled`
```suggestion
  "queryName": "AKS Private Cluster Disabled",
```